### DISCUSSION TITLE
Getting a ban when using swiftagram

### DISCUSSION THREAD

**Discussion by ramimoghrabi** [sbertix/Swiftagram#261] [General]
Is there a way we can do to stop getting banned when using this lib, i noticed that when sending more requests the account will get banned for 24 hours generally, how can we workaround this issue ? Thanks for the help in advance

**Comment by sbertix** [comment]
Hey @ramimoghrabi, 

Unfortunately there's not a "proper" way to deal with this. 
Instagram Private API endpoints are called by the app always after user interaction, so I can't suggest nothing more than possibly be a little more cautious with how many calls are executed through your client. 
Apologies

**Reply by ramimoghrabi** [reply]
Ah ok, thanks @sbertix, so you suggest that i limit calls to when it is necessary to fetch. But do you have any estimation on the call limit so that i can make it accurate? If not it is ok, i will check too.
